src/Popper.js

```diff
@@ -79,7 +79,7 @@ export class InnerPopper extends React.Component<PopperProps, PopperState> {
     safeInvoke(this.props.innerRef, popperNode);
     this.popperNode = popperNode;
 
-    if (!this.popperInstance) this.updatePopperInstance();
+    this.updatePopperInstance();
```

Comment:
What's the point of updating a non existing popper instance?

Reply:
So there should rather be a check for if `popperNode` is not defined than?

Reply:
Sorry, misunderstood. Doesn't that line of code originally mean that the popper instance is only updated if it's non existing?

Reply:
@FezVrasta before this fix, on L80 the `popperNode` gets updated on the class, but not inside the `popperInstance` which leaves the class in an inconsistent state. when the `popperNode` changes, we need to reflect that in the `popperInstance` as well, so it's necessary to call `updatePopperInstance()` (which is actually a misleading name since it updates the instance).

right now updating `popperNode` does not work when the popperInstance has already been created.